events/events.go

```diff
@@ -361,6 +366,41 @@ type OperationDeprovisionedData struct {
 	Team   *Team        `json:"team,omitempty"`
 }
 
+// OperationResized represents a resize operation event.
+type OperationResized struct {
+	BaseBody
+	Data OperationResizedData `json:"data"`
+}
+
+// OperationResizedData holds the event specific data.
+type OperationResizedData struct {
+	OperationID manifold.ID `json:"operation_id"`
+
+	ResourceID manifold.ID `json:"resource_id"`
+	Resource   *Resource   `json:"resource,omitempty"`
+
+	UserID *manifold.ID `json:"user_id,omitempty"`
+	User   *User        `json:"user,omitempty"`
+
+	TeamID *manifold.ID `json:"team_id,omitempty"`
+	Team   *Team        `json:"team,omitempty"`
+
+	ProjectID *manifold.ID `json:"project_id,omitempty"`
+	Project   *Project     `json:"project,omitempty"`
+
+	ProviderID *manifold.ID `json:"provider_id,omitempty"`
```

Comment:
Why is this optional?

Reply:
It is optional because during the resize we only have the new plan id available. We get the provider, product and region after expansion. It could be required, but it would start empty.

Reply:
ahk cool :+1: